services/web/server/src/simcore_service_webserver/products/_model.py

```diff
@@ -232,14 +232,14 @@ def to_statics(self) -> dict[str, Any]:
         # keys will be named as e.g. displayName, supportEmail, ...
         return self.dict(
             include={
-                "display_name",
-                "support_email",
-                "vendor",
-                "issues",
-                "manuals",
-                "support",
-                "is_payment_enabled",
-                "is_dynamic_services_telemetry_enabled",
+                "display_name": True,
```

Comment:
NOTE: there is no need to do this. `dict`s are only necessary when you add embedded fields as `foo.bar.zoo`